models/repo.go

```diff
@@ -197,6 +206,16 @@ func (repo *Repository) ColorFormat(s fmt.State) {
 		repo.Name)
 }
 
+// IsBeingMigrated indicates that repository is being migtated
+func (repo *Repository) IsBeingMigrated() bool {
+	return repo.Status == RepositoryBeingMigrated
+}
+
+// IsBeingCreated indicates that repository is being migtated or forked
+func (repo *Repository) IsBeingCreated() bool {
+	return repo.IsBeingMigrated()
```

Comment:
This doesn't seem to act differently for forked repositories.

Reply:
This implementation is reserved. When we add a new beingforked status, we could change this method.